src/Redux/Actions/login/loginAction.js

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import {
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+  LOGIN_FAIL
+} from "../../ActionTypes/login/loginActionTypes";
+
+const basePath = "https://champs-bn-api.herokuapp.com/api/v1";
+
+export const loginRequest = () => ({
+  type: LOGIN_REQUEST
+});
+
+export const loginSuccess = payload => ({
+  type: LOGIN_SUCCESS,
+  payload
+});
+
+export const loginFail = payload => ({
+  type: LOGIN_FAIL,
+  payload
+});
```

Comment:
Leave an empty space after the semi-colon.

Reply:
Feedback implemented

Reply:
Not yet.